.htaccess

```diff
@@ -3,6 +3,8 @@ RewriteEngine On
 RewriteRule ^numpy(/.*) https://numpy.org/doc/stable$1 [R=permanent]
 # See https://github.com/scipy/scipy/issues/14267
 Redirect 301 /scipy/reference/objects.inv /scipy/objects.inv
+RewriteRule scipy/ scipy-1.8.0/reference
```

Comment:
I think this needs to be as above for scipy: `^scipy(/.*) scipy-1.8.0/html-scipyorg` should work

Reply:
And probably also one for 1.8:

```
^scipy-1.8.0(/.*) scipy-1.8.0/html-scipyorg$1
```

Reply:
@maxnoe I applied and deployed your suggestions from the review process here.

Note that this did break someting that worked before--the HTML zip and PDF links are now broken on the side list for `1.8.0`:

![image](https://user-images.githubusercontent.com/7903078/152850789-c93d968b-a13c-4359-a466-04e65415e862.png)

Reply:
Sorry, I think the regex matches to much,  the slash should be in front of the parentheses and in front of $1

Reply:
The adjusted regex would still catch the tarball/PDF, which haven't been moved to the new location with the new theme. There's another issue though noted in https://github.com/tylerjereddy/htaccess_backup/pull/1#issuecomment-1031802785 which I don't understand yet, since the file seems to be there.

Reply:
Rewrite rule are processed in order, so of we add one first matching exactly the pdf and zipped html  that should work, right?